From 9d7c6b85d0d2f8e26298cbaf072491662a9d2702 Mon Sep 17 00:00:00 2001
From: Pauli Virtanen <pav@iki.fi>
Date: Sat, 6 Apr 2024 21:59:16 +0300
Subject: [PATCH] monitors/bluez: fix BAP device set node naming

It is intended that its name is the actual device node name.
Fix this, as it was broken in some recent changes.
---
 src/scripts/monitors/bluez.lua | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/src/scripts/monitors/bluez.lua b/src/scripts/monitors/bluez.lua
index 336cd619..c3c00c60 100644
--- a/src/scripts/monitors/bluez.lua
+++ b/src/scripts/monitors/bluez.lua
@@ -171,7 +171,6 @@ end)
 
 function createSetNode(parent, id, type, factory, properties)
   local args = {}
-  local name
   local target_class
   local stream_class
   local rules = {}
@@ -181,12 +180,10 @@ function createSetNode(parent, id, type, factory, properties)
   local channels = channels_json:parse ()
 
   if properties["media.class"] == "Audio/Sink" then
-    name = "bluez_output_internal"
     args["combine.mode"] = "sink"
     target_class = "Audio/Sink/Internal"
     stream_class = "Stream/Output/Audio/Internal"
   else
-    name = "bluez_input_internal"
     args["combine.mode"] = "source"
     target_class = "Audio/Source/Internal"
     stream_class = "Stream/Input/Audio/Internal"
@@ -214,7 +211,6 @@ function createSetNode(parent, id, type, factory, properties)
     )
   end
 
-  properties["node.name"] = name
   properties["node.virtual"] = false
   properties["device.api"] = "bluez5"
   properties["api.bluez5.set.members"] = nil
-- 
GitLab

